cfgmgr/vxlanmgr.h

```diff
@@ -49,28 +80,40 @@ class VxlanMgr : public Orch
     */
     bool isVrfStateOk(const std::string & vrfName);
     bool isVxlanStateOk(const std::string & vxlanName);
+    bool isVlanStateOk(const string &vlanName);
```

Comment:
`string` miss to add std namespace `std::string`

Reply:
done.